config/install/media_entity.bundle.video.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - video_embed_media
+id: video
+label: Video
+description: 'Use Video for embedding videos hosted by YouTube, Vimeo, or some other provider.'
+type: video_embed_field
+queue_thumbnail_downloads: false
+new_revision: false
+type_configuration:
+  source_field: field_video
+field_map: {  }
```

Comment:
Would be nice to map "Video ID" and "Video source human name" to entity fields. We need to add them of course.